server/v2/api/grpcgateway/interceptor.go

```
@@ -41,57 +59,66 @@
+		return
+	}
+
+	// msg population based on http method.
+	var inputMsg gogoproto.Message
+	var err error
 	switch request.Method {
-	case http.MethodPost:
-		msg, err = createMessageFromJSON(match, request)
 	case http.MethodGet:
-		msg, err = createMessage(match)
+		inputMsg, err = g.createMessageFromGetRequest(request.Context(), in, request, msg, match.Params)
```

golangci-lint (1.62.2)

SA4006: this value of `err` is never used

(staticcheck)